### DISCUSSION TITLE
No 'Execute SQL' option?

### DISCUSSION THREAD

**Discussion by skinnyshiro** [sqlitebrowser/sqlitebrowser#3300] [Q&A] [answered]
Hi Everyone,
Hoping somebody might be able to help out. I have been unable to find any other threads about this problem, but in my DB Browser there is no 'Execute SQL' tab for me to open. Regarding the database file, it is a publicly available research database and I'm able to run SQLite code against it in Python, so in theory it should function. Any help or guidance as to what I may be doing wrong would be greatly appreciated. 

![image](https://user-images.githubusercontent.com/74924673/226199922-cbf7c157-8e91-40ca-8f41-5d7ffc66299a.png)

**Comment by justinclift** [comment, marked as answer]
As a first guess, check in the **View** menu to see if "Browse Data" has been turned off.

If it has, click the toggle switch to turn it back on. :smile:

![image](https://user-images.githubusercontent.com/406299/226202897-a098ff6a-797d-4f8d-898e-1eddfec75265.png)

**Reply by skinnyshiro** [reply]
I'm both embarrassed and glad to say that this was in fact the problem. Thank you for the quick response!

**Reply by justinclift** [reply]
All good. :smile:
